app/dashboard/views.py

```diff
@@ -522,6 +522,11 @@ def send_tip_2(request):
 
 def onboard(request, flow):
     """Handle displaying the first time user experience flow."""
+
+    if not request.user or not request.user.is_authenticated:
```

Comment:
This is wrong. This will cause anyone who goes to the onboarding flow to be redirected to the github login. That is not expected. We want this only for the standalone avatar page. So this check should be only if someone is trying to access endpoints like: `/onboard/contributor/?steps=avatar` or `/onboard/profile` and so on.